### PR TITLE
Mark the OG and CFP as being closed

### PR DESCRIPTION
**Pull request by drewbrew**
## Description

Removes the links to submit to the CFP/OGs from the home page, speaking page, and OG page

### CODE REVIEW

**Comment by drewbrew** [comment]
image

image


image

File: _pages/opportunity-grants.md [hunk 1]
```diff
@@ -42,7 +43,7 @@ We feel the wording "opportunity grants" makes a positive shift towards being a
 
 ### When will I find out whether I've received an opportunity grant?
 
-You will be notified by June 14, 2024.
+You will be notified by July 8, 2024.
```
Comment (tim-schilling) [inline _pages/opportunity-grants.md:46]: There's a mention above about June 14th. Is that correct or do we need to make these consistent?

Comment (drewbrew) [reply _pages/opportunity-grants.md:46]: Good catch. We should update the other one too.

Comment (drewbrew) [reply _pages/opportunity-grants.md:46]: that's done

**Review by tim-schilling — APPROVED** [review]
Thanks Drew!

**Comment by tim-schilling** [comment]
For what it's worth, I also checked pretalx and the OG google form. Both already say they are closed so we're all good there too.

**Pull request merged**